src/plugin.js

```diff
@@ -41,6 +55,19 @@
     }
   }
 
+  Object.keys(backfillClasses).forEach(function(className) {
+    backfillClasses[className].forEach(function(name) {
+      var Component = videojs.getComponent(name);
+      var createEl = Component.prototype.createEl;
+
+      Component.prototype.createEl = function() {
```

Comment:
If you guys both feel this is the right approach, it's cool with me. It's a pretty fundamental method to monkey-patch though.